projects/1.20-fabric/assets/mythicmetals/mythicmetals/lang/zh_cn.json

```
@@ -0,0 +1,1152 @@
+	"tag.item.c.metallurgium_blocks": "炼金块",
+	"tag.item.c.metallurgium_ingots": "炼金锭",
+	"tag.item.c.midas_gold_blocks": "迈达斯金块",
+	"tag.item.c.midas_gold_ingots": "迈达斯金锭",
+	"tag.item.c.midas_gold_ores": "迈达斯金矿石",
+	"tag.item.c.morkite": "墨菱石",
+	"tag.item.c.morkite_blocks": "墨菱石块",
+	"tag.item.c.morkite_ores": "墨菱矿石",
+	"tag.item.c.mythril_blocks": "秘银块",
+	"tag.item.c.mythril_ingots": "秘银锭",
+	"tag.item.c.mythril_ores": "秘银矿石",
+	"tag.item.c.orichalcum_blocks": "山铜块",
+	"tag.item.c.orichalcum_ingots": "山铜锭",
+	"tag.item.c.orichalcum_ores": "山铜矿石",
+	"tag.item.c.osmium_blocks": "锇块",
+	"tag.item.c.osmium_ingots": "锇锭",
+	"tag.item.c.osmium_ores": "锇矿石",
+	"tag.item.c.palladium_blocks": "钯金块",
+	"tag.item.c.palladium_ores": "钯金矿石",
+	"tag.item.c.platinum_blocks": "铂金块",
+	"tag.item.c.platinum_ingots": "铂金锭",
+	"tag.item.c.platinum_ores": "铂金矿石",
+	"tag.item.mythicmetals.kyber_blocks": "凯伯块",
+	"tag.item.mythicmetals.kyber_gear": "凯伯装备",
+	"tag.item.mythicmetals.kyber_ingots": "凯伯锭",
+	"tag.item.mythicmetals.kyber_ores": "凯伯矿石",
+	"tag.item.mythicmetals.manganese_blocks": "锰块",
+	"tag.item.mythicmetals.manganese_ingots": "锰锭",
+	"tag.item.mythicmetals.manganese_ores": "锰矿石",
+	"tag.item.mythicmetals.metallurgium_ingots": "炼金锭",
+	"tag.item.mythicmetals.metals": "金属",
+	"tag.item.mythicmetals.midas_gold_armor": "迈达斯金盔甲",
+	"tag.item.mythicmetals.midas_gold_blocks": "迈达斯金块",
+	"tag.item.mythicmetals.midas_gold_ingots": "迈达斯金锭",
+	"tag.item.mythicmetals.midas_gold_ores": "迈达斯金矿石",
+	"tag.item.mythicmetals.morkite": "墨菱石",
+	"tag.item.mythicmetals.morkite_blocks": "墨菱石块",
+	"tag.item.mythicmetals.morkite_ores": "墨菱矿石",
+	"tag.item.mythicmetals.mythril_blocks": "秘银块",
+	"tag.item.mythicmetals.mythril_gear": "秘银装备",
+	"tag.item.mythicmetals.mythril_ingots": "秘银锭",
+	"tag.item.mythicmetals.mythril_ores": "秘银矿石",
+	"tag.item.mythicmetals.orichalcum_gear": "山铜装备",
+	"tag.item.mythicmetals.orichalcum_ingots": "山铜锭",
+	"tag.item.mythicmetals.orichalcum_ores": "山铜矿石",
+	"tag.item.mythicmetals.osmium_blocks": "锇块",
+	"tag.item.mythicmetals.osmium_gear": "锇装备",
+	"tag.item.mythicmetals.osmium_ingots": "锇锭",
+	"tag.item.mythicmetals.osmium_ores": "锇矿石",
+	"tag.item.mythicmetals.palladium_armor": "钯金盔甲",
+	"tag.item.mythicmetals.palladium_blocks": "钯金块",
+	"tag.item.mythicmetals.palladium_gear": "钯金装备",
+	"tag.item.mythicmetals.palladium_ingots": "钯金锭",
+	"tag.item.mythicmetals.palladium_ores": "钯金矿石",
+	"tag.item.mythicmetals.platinum_blocks": "铂金块",
+	"tag.item.mythicmetals.platinum_ingots": "铂金锭",
+	"tag.item.mythicmetals.platinum_ores": "铂金矿石",
+	"tag.item.mythicmetals.armor": "神话金属盔甲",
+	"tag.item.mythicmetals.tools": "神话金属工具",
+	"subtitles.mythicmetals.equip_metallurgium": "炼金盔甲：铿锵",
+	"subtitles.mythicmetals.equip_midas_gold": "迈达斯金盔甲：叮当",
+	"advancement.mythicmetals.carmot_template.title": "用了时运还是搜刮大法？",
+	"tooltip.midas_gold.level.29": "你得往后让让了……",
+	"tooltip.midas_gold.level.20": "你的剑中已蕴含了迈达斯国王的无上力量",
+	"tooltip.midas_gold.level.33": "你说是个无名小卒虾炒的饭……？",
```

```suggestion
	"tooltip.midas_gold.level.33": "你说是个虾炒的饭……？",
```
大概是某个shrimp fried rice的梗视频吧